data/seeds/production/01_users.js

```diff
@@ -2,89 +2,72 @@ exports.seed = function(knex, Promise) {
   // Deletes ALL existing entries
   return knex('users')
     .del()
-    .then(function() {
+    .then(async function() {
       // Inserts seed entries
-      return knex('users').insert([
+      const providerAccounts = [];
+      const providers = await knex('providers');
+      for (let i = 0; i < providers.length; i++) {
+        const name = providers[i].name.split(' ');
+        providerAccounts.push({
+          username: `${name[0]}_${name[1]}`,
+          password: Math.random().toString().slice(3, 19),
+          email: `${name[0]}@email.com`,
+          providerId: providers[i].id,
+          createdAt: knex.fn.now(),
+        });
+      }
+      const nonProviders = [
         {
-          id: 1,
-          username: 'edreeseg',
+          username: 'ed_reeseg',
           password: 'test123',
```

Comment:
Is it possible to log in as seed users despite the passwords here not being hashed?

Reply:
At the moment it's not possible, but I'm probably going to run the seeds' passwords through a function that'll run bcrypt on them to hash, to make logging in possible.